models/tensorflow_hub.py

```diff
@@ -0,0 +1,26 @@
+"""
+Details layer-based transfer learning models from TensorFlow Hub.
+Note that using these models may require configuring specific image sizes in `configs/model.py`.
+"""
+
+from core.transfer_learning import LayerBasedTransferLearningModel
+
+
+class InceptionV3(LayerBasedTransferLearningModel):
+    handle = 'https://tfhub.dev/google/imagenet/inception_v3/feature_vector/5'
```

Comment:
Was this the one used for 299, 92% or was that V2?

Reply:
Yep that's the one